media-sound/musescore/files/musescore-2.0.3-fix-buildsystem.patch

```diff
@@ -0,0 +1,473 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 2c4fe57..915e00b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -147,23 +147,10 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
+       endif()
+ endif ("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
+ 
+-if (APPLE)
+-   set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -fPIC -stdlib=libc++ -g -Wno-inconsistent-missing-override")
+-   set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -fPIC -stdlib=libc++ -O2 -DNDEBUG -DQT_NO_DEBUG -Wno-inconsistent-missing-override")
+-else (APPLE)
+-   if (MINGW)
+-      # -mno-ms-bitfields see #22048
+-      set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -mno-ms-bitfields -g")
+-      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -mno-ms-bitfields -O2 -DNDEBUG -DQT_NO_DEBUG")
+-      set(CMAKE_EXE_LINKER_FLAGS "-Wl,--large-address-aware")
+-   else (MINGW)
+-      set(CMAKE_CXX_FLAGS_DEBUG   "-std=c++11 -fPIC -fPIE -g")
+-      set(CMAKE_CXX_FLAGS_RELEASE "-std=c++11 -fPIC -O2 -DNDEBUG -DQT_NO_DEBUG")
+-   endif (MINGW)
+-endif(APPLE)
++set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fPIC -fPIE -DNDEBUG -DQT_NO_DEBUG")
```

Comment:
I originally patched the build system, but would like to make it less dependent on hardcoding `-std=c++11`. Can you try whether the thing still works with
```
set(CMAKE_CXX_STANDARD 11)
set(CMAKE_CXX_STANDARD_REQUIRED ON)
set(CMAKE_CXX_EXTENSIONS OFF)
```
and if so, use that? If it does, remove all mentions of `-std=c++11`. It would be great if we could then try to upstream this patch.